admin/c2cgeoportal_admin/__init__.py

```diff
@@ -66,4 +66,6 @@ def includeme(config: Configurator):
     config.include('c2cgeoform')
     config.include('c2cgeoportal_commons')
     config.include('c2cgeoportal_admin.routes')
+    # use pyramid_tm to hook the transaction lifecycle to the request
+    config.include('pyramid_tm')
```

Comment:
'pyramid_tm' included twice in case of c2cgeoportal_admin.main

Reply:
Thanks, fixed